### PR TITLE
Add mapper function for posting to back-end.

### PR DESCRIPTION
**Pull request by awong-dev**
Gigantic check-in.

Still needs fixing and several items (mentioned in the initial commented
form store section) require input from ES team.

### CODE REVIEW

**Comment by alexose** [comment]
👍 

This looks totally reasonable at first glance, but it's hard to know whether or not it works without something that mocks the external API.

**Comment by awong-dev** [comment]
@alexose This is @joesoftware 's PR.  I would work with him.. In general we're following the rule though that if someone LGTM's a PR, they should merge it.

However,I don't think Joe actually meant for this to be reviewed quite yet...

**Comment by alexose** [comment]
Okay, I'll hold off for now!

File: src/client/utils/mapper.js [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+      return 'W';
+    case 'Divorced':
+      return 'D';
+    default:
+      return 'U';
+  }
+}
+
+function makeSpouse(nameAndGeneralInformation, spouseInformation) {
+  if (nameAndGeneralInformation.maritalStatus !== 'Never Married') {
```
Comment (robbiethegeek) [inline src/client/utils/mapper.js]: why not do 
if (nameAndGeneralInformation.maritalStatus === 'Never Married) {
  return undefined; // also why undefined? is there a reason?
}
return {....

**Comment by joesoftware** [comment]
New form store is in flux, but this is being merged so someone can pickup the work.

**Pull request merged**